cmd/ocm/create/cluster/cmd.go

```diff
@@ -377,6 +394,68 @@ func getVersionOptionsWithDefault(connection *sdk.Connection, channelGroup strin
 	return
 }
 
+func getSubscriptionTypeOptions(connection *sdk.Connection) ([]arguments.Option, error) {
+	options := []arguments.Option{}
+	billingModels, err := getBillingModels(connection)
+	if err != nil {
+		return options, err
+	}
+	for _, billingModel := range billingModels {
+		//Standard billing model should always be the first option
```

Comment:
Can we refactor/simplify this?

Reply:
Simplified by using a function to create the option and reusing outside the if else block

Reply:
I think we can simplify further...

How about this:

```
		if billingModel.ID() == billing.StandardSubscriptionType {
			options = append([]arguments.Option{option}, options...)
		} else {
			options = append(options, option)
		}
```